core/config/locales/en.yml

```
@@ -1539,12 +1539,12 @@ en:
       authorized: "Authorized"
       captured: "Used"
       allocated: "Added"
-      apply: Apply Store Credit
-      remove: Remove Store Credit
-      applicable_amount: "%{amount} in store credit will be applied to this order."
-      available_amount: "You have %{amount} in Store Credit available!"
-      remaining_amount: "You have %{amount} remaining in your account's Store Credit."
-      additional_payment_needed: Select another payment method for the remaining %{amount}.
+      apply: Apply
+      remove: Remove
+      applicable_amount: "<b>%{amount}</b> in store credit will be applied to this order."
```

```suggestion
      applicable_amount: "<strong>%{amount}</strong> in store credit will be applied to this order."
```

let's use `strong` tag, not `b`